ui/src/literature/components/__tests__/__snapshots__/CiteModalAction.test.jsx.snap

```diff
@@ -31,7 +31,7 @@ exports[`CiteModalAction renders with all props 1`] = `
     title="Cite Article"
     transitionName="zoom"
     visible={false}
-    width="50%"
+    width={520}
```

Comment:
why we have to change this?

Reply:
I realized that it was crap on mobile, it's better to use antd default size, which is overriden automatically on mobile.